src/System.Private.Xml/src/System/Xml/Serialization/CodeGenerationoptions.cs

```diff
@@ -2,7 +2,13 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#if XMLSERIALIZERGENERATOR
+using System;
```

Comment:
For XMLSERIALIZERGENERATOR, we need `using System` since the new namespace, `Microsoft.XmlSerializer.Generator`, is not under `System.XXX`. But I think we can put the `using` outside `#if`.